source/stylesheets/refills/_flashes.scss

```diff
@@ -1,40 +1,28 @@
 $base-spacing: 1.5em !default;
-$alert-color: #fff6bf !default;
-$error-color: #fbe3e4 !default;
-$notice-color: #e5edf8 !default;
-$success-color: #e6efc2 !default;
+$flashes: (
+  alert: #fff6bf !default,
```

Comment:
Interesting! It appears you can only set the `!default` flag on standard variables, and not within a map: http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_